build-aux/com.github.melix99.telegrand.Devel.json

```diff
@@ -110,6 +110,22 @@
                 }
             ]
         },
+        {
+            "name": "libshumate",
```

Comment:
The new libshumate dependency should be listed in the README.

Reply:
Good catch! Same with meson-build file.